respy/simulate.py

```diff
@@ -42,12 +42,18 @@ def simulate(params, options):
         options["simulation_seed"],
     )
 
-    simulated_data = simulate_data(state_space, base_draws_sims, optim_paras, options)
+    base_draws_wage = create_base_draws(
+        base_draws_sims.shape, seed=options["simulation_seed"] + 1
+    )
+
+    simulated_data = simulate_data(
+        state_space, base_draws_sims, base_draws_wage, optim_paras, options
+    )
 
     return state_space, simulated_data
 
 
-def simulate_data(state_space, base_draws_sims, optim_paras, options):
+def simulate_data(state_space, base_draws_sims, base_draws_wage, optim_paras, options):
```

Comment:
You add base_draws_wage but never use it.

Reply:
Is fixed now.